include/nbl/video/IGPUBuffer.h

```diff
@@ -27,18 +27,18 @@ class IGPUBuffer : public asset::IBuffer, public IDriverMemoryBacked, public IBa
     public:
 		struct SCachedCreationParams
 		{
+			size_t declaredSize = 0ull;
 			core::bitflag<E_USAGE_FLAGS> usage = EUF_NONE;
-			bool canUpdateSubRange = false; // whether `IGPUCommandBuffer::updateBuffer` can be used on this buffer
 			asset::E_SHARING_MODE sharingMode = asset::ESM_EXCLUSIVE;
-			size_t declaredSize = 0ull;
+			bool canUpdateSubRange = false; // whether `IGPUCommandBuffer::updateBuffer` can be used on this buffer
```

Comment:
can you somehow make sure and static-assert `sizeof(SCachedCreationParams)==16` ?